setup.py

```diff
@@ -251,7 +252,7 @@ def finalize_options(self):
     'testfixtures',
     'Morfessor==2.0.2a4',
     'python-Levenshtein >= 0.10.2',
-    'visdom >= 0.1.8, != 0.1.8.7',
```

Comment:
Why these comma changes?

-1 from me: they make diffs unnecessarily complicated, when adding another list item. Better with the comma (also more consistent, less room for typos).